trade-finance-manager-api/src/v1/api.js

```diff
@@ -501,9 +502,7 @@ const getFunctionsAPI = async (type = 'ACBS', url = '') => {
     return response.data;
   } catch (err) {
     console.error(`Unable to getFunctionsAPI for ${modifiedUrl}`, { err });
-    return err.response.data
-      ? err.response.data
-      : err;
+    return err?.response?.data ? err?.response?.data : err;
```

Comment:
Second `err?.response?.data` can be changed to `err.response.data`

Reply:
done